app/containers/AddressChooserModalContainer.tsx

```diff
@@ -42,15 +45,12 @@ export class AddressChooserModalContainer extends React.Component<
       addresses,
       loading: true,
     };
+  }
 
+  public componentDidMount() {
     this.obtainData().catch(error => {
       throw new Error(error);
```

Comment:
this catch and throw doesnt really do anything isn't it? Its just another rejected, uncaought promise